apps/integration_tests/selenium_generic.py

```diff
@@ -64,7 +64,7 @@ def setup_method(self, method):
         opt.add_argument("--disable-popup-blocking")
         opt.add_argument("--enable-javascript")
         opt.add_argument('--allow-insecure-localhost')
-        opt.add_argument('--window-size=1920,1080')
+        # opt.add_argument('--window-size=1920,1080')
```

Comment:
remove

Reply:
this one: I'll restore it since there is a ticket to remove that line, I prefer get that done when that ticket is worked on...